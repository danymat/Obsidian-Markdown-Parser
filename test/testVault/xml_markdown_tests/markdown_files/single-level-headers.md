root section

# header 1

section 1

# header 2

section 2

# header 3

section 3
