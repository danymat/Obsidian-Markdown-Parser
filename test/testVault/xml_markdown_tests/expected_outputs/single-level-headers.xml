<root level="0">root section

<section level="1" title="header 1"># header 1

section 1

</section><section level="1" title="header 2"># header 2

section 2

</section><section level="1" title="header 3"># header 3

section 3
</section></root>
